bundles/ApplicationLoggerBundle/src/Maintenance/LogArchiveTask.php

```diff
@@ -63,6 +64,19 @@ public function execute(): void
         $sql = 'SELECT %s FROM '.ApplicationLoggerDb::TABLE_NAME.' WHERE `timestamp` < DATE_SUB(FROM_UNIXTIME('.$timestamp.'), INTERVAL '.$archive_threshold.' DAY)';
 
         if ($db->fetchOne(sprintf($sql, 'COUNT(*)')) > 0) {
+
+            $storageEngine = $this->config['applicationlog']['archive_db_table_storage_engine'];
```

Comment:
I believe checking table exists via schemaManager would be less expensive (and table-lock safer?) than checking the symfony config, fetching `SHOW engines` and `CREATE TABLE IF NOT EXISTS` everytime this method would be executed.

Suggesting to do something like
```suggestion
           if (!$db->createSchemaManager()->tablesExist($tablename)) {
               $storageEngine = $this->config['applicationlog']['archive_db_table_storage_engine'];
```

and replace
`$db->executeQuery('CREATE TABLE IF NOT EXISTS '.$tablename." (` with just `$db->executeQuery('CREATE TABLE  '.$tablename." (` within the !tablesExist check